web/src/app/modules/shared/services/preferences/preferences.service.ts

```diff
@@ -104,83 +144,173 @@ export class PreferencesService implements OnDestroy {
+      ? [this.getGeneralPanels(), this.getDeveloperPanels()]
+      : [this.getGeneralPanels()];
+
     return {
       updateName: 'generalPreferences',
-      panels: [
+      panels,
+    };
+  }
+
+  private getDeveloperPanels(): PreferencePanel {
+    return {
+      name: 'Development',
+      sections: [
         {
-          name: 'General',
-          sections: [
+          name: 'Frontend Source',
+          elements: [
             {
-              name: 'Color Theme',
-              elements: [
-                {
-                  name: 'general.theme',
-                  type: 'radio',
-                  value: this.themeService.isLightThemeEnabled()
-                    ? 'light'
-                    : 'dark',
-                  config: {
-                    values: [
-                      {
-                        label: 'Dark',
-                        value: 'dark',
-                      },
-                      {
-                        label: 'Light',
-                        value: 'light',
-                      },
-                    ],
+              name: 'development.embedded',
+              type: 'radio',
+              value: this.embedded.value ? 'embedded' : 'proxied',
+              config: {
+                values: [
+                  {
+                    label: 'Embedded',
+                    value: 'embedded',
                   },
-                },
-              ],
+                  {
+                    label: 'Proxied',
+                    value: 'proxied',
+                  },
+                ],
+              },
             },
             {
-              name: 'Navigation',
-              elements: [
+              name: 'development.frontendUrl',
+              type: 'text',
+              value: this.frontendUrl.value,
+              disableConditions: [
                 {
-                  name: 'general.navigation',
-                  type: 'radio',
-                  value: this.navCollapsed.value ? 'collapsed' : 'expanded',
-                  config: {
-                    values: [
-                      {
-                        label: 'Expanded',
-                        value: 'expanded',
-                      },
-                      {
-                        label: 'Collapsed',
-                        value: 'collapsed',
-                      },
-                    ],
-                  },
+                  lhs: 'development.embedded',
+                  op: Operation.Equal,
+                  rhs: 'proxied',
                 },
               ],
+              config: {
+                label: 'Frontend Proxy Url',
+                placeholder: 'http://example.com',
+              },
+            },
+          ],
+        },
+        {
+          name: 'Logging verbosity (requires restart)',
+          elements: [
+            {
+              name: 'development.verbose',
+              type: 'radio',
+              value: this.verbose.value ? 'high' : 'low',
```

Comment:
normal / debug for these and the labels.

Reply:
✅